geowebcache/core/src/main/java/org/geowebcache/mime/MimeType.java

```diff
@@ -198,7 +198,8 @@ public int hashCode() {
 
 
     public boolean isCompatible(String otherMimeType) {
-        return mimeType.equalsIgnoreCase(otherMimeType) || otherMimeType.startsWith(otherMimeType);
+        return mimeType.equalsIgnoreCase(otherMimeType) ||
+            otherMimeType.startsWith(mimeType);
```

Comment:
It looks like this makes it case insensitive if it's otherwise an exact match, but case sensitive if matching a subtype, which is really an odd thing to do.